packages/go/index.ts

```diff
@@ -456,20 +454,10 @@ Learn more: https://vercel.com/docs/runtimes#official-runtimes/go
     environment: {},
   });
 
-  const watch = parsedAnalyzed.watch;
-  let watchSub: string[] = [];
-  // if `entrypoint` located in subdirectory
-  // we will need to concat it with return watch array
-  if (entrypointArr.length > 1) {
-    entrypointArr.pop();
```

Comment:
This mutates `entrypointArr`. Is it okay to remove it? I can see `entrypointArr` is used below. 🤔